content/en/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md

```diff
@@ -180,7 +180,23 @@ for database debugging.
     PONG
     ```
 
+### Optionaly let _kubectl_ choose the local port
 
+If you don't need a specific local port, you can let `kubectl` choose and allocate 
+the local port and thus relieve you from having to manage local port conflicts, with 
+the slightly simpler syntax:
+
+```shell
+kubectl port-forward deployment/redis-master :6379
+```
+
+An allocated local port in the upper range will be displayed as a result, the output 
+will be similar to this:
```

Comment:
How about
```suggestion
The `kubectl` tool finds a local port number that is not in use (avoiding low ports numbers,
because these might be used by other applications). The output is similar to:
```

Reply:
Well put, english is not my first language so thanks for the taking the time to rephrase. 👍